### ISSUE TITLE
Azure CLI running in Powershell becomes unresponsive

### ISSUE THREAD

**Issue by PeiwenGaoMS** [Azure/azure-cli#23539]
> ### `az feedback` auto-generates most of the information requested below, as of CLI version 2.0.62

## **Related command**

All az command lines do not work.

## **Describe the bug**

I recently faced a tricky problem with the azure cli that I run any “az…” command in powershell and no respond, like the image below. 
![image](https://user-images.githubusercontent.com/111329184/184785817-4df38e62-4b8f-4eeb-8c4d-901793877b55.png)
At first, I thought that there was a problem because anaconda was installed. Later, I uninstalled anaconda and found that it was still the same. I don't know where the problem is. I have tried to see if there is anyone with a similar problem to me but I have not found. I've also tried uninstalling or repairing the Azure CLI, and reinstalling a different or the same version(2.38.0 or 2.39.0), but it didn't work.

## **To Reproduce**

I can't help you reproduce, because only my computer has such a problem, my colleagues are normal. I had this problem suddenly.

## **Expected behavior**

I want all az command line to work fine.

## **Environment summary**

**Install Method:** MSI
**Version:** 2.38.0 or 2.39.0
**OS version:** Windows11
**Shell Type:** Powershell

**Comment by yonzhan** [comment]
@jiasli for awareness

**Comment by bebound** [comment]
What's the output of `(gcm az).Source` in PowerShell?
You may need to delete this azure-cli package.

**Comment by PeiwenGaoMS** [comment]
> What's the output of `(gcm az).Source` in PowerShell? You may need to delete this azure-cli package.

It's strange that this command displays "fatal" in **Powershell**, but a path is displayed in **Anaconda Powershell Prompt** , so it still has something to do with downloading Anaconda? 

- Powershell:

![image](https://user-images.githubusercontent.com/111329184/185025095-3ec1a37f-ade3-4a43-a9ef-b0c74cccb5da.png)

- Anaconda Powershell Prompt:

![image](https://user-images.githubusercontent.com/111329184/185025068-5d335c23-77f3-4fe5-a9f2-1dd3030a0352.png)

By the way, which package do you say should be removed? Is the `C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2`?

**Comment by bebound** [comment]
Does `az` works on Anaconda PowerShell or cmd.exe?

Looks like cli is successfully installed but you PowerShell is down.

**Comment by PeiwenGaoMS** [comment]
> Does `az` works on Anaconda PowerShell or cmd.exe?
> Looks like cli is successfully installed but your PowerShell is down.
At first, I couldn't open cmd.exe. Then, I encountered a similar problem to me. It was also because of frequent installation and uninstallation of Anaconda that cmd.exe could not be opened. So I followed the following method.
![image](https://user-images.githubusercontent.com/111329184/185068747-27a0738a-e38b-4329-bf9b-14b5d437ed71.png)
Now not only can cmd.exe be opened, all the az commands can also work fine too.
